patches/ios-build-bots-scripts-test_runner.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ios/build/bots/scripts/test_runner.py b/ios/build/bots/scripts/test_runner.py
+index 46a8c2805f7bd19036f08882b7dcbdf5097da6d4..28c95f4e44db49da608279ca4016a4c844edaf6d 100644
+--- a/ios/build/bots/scripts/test_runner.py
++++ b/ios/build/bots/scripts/test_runner.py
+@@ -376,6 +376,7 @@ class TestRunner(object):
+   @staticmethod
+   def remove_proxy_settings():
+     """removes any proxy settings which may remain from a previous run."""
++    return
```

Comment:
is this script used somehow? if yes, pls consider fixing this via chromium_src override.

Reply:
this is called by `run.py` but fails because it requires admin. Do we have some docs on overriding python files because I'm not sure what to do here that could just be an override and not a patch

Reply:
I can't actually do this because I can't call run.py with out python path so overrides don't work